docs/user-guide/cli-using-creating-profiles.md

```diff
@@ -0,0 +1,246 @@
+                    "properties": {
+                        "port": 1443
+                    }
+                }
+            }
+        },
+        "base": {
+            "type": "base",
+            "properties": {
+                "rejectUnauthorized": true
+            },
+            "secure": [
+                "user",
+                "password"
+            ]
+        }
+    },
+    "defaults": {
+        "zosmf": "lpar2.zosmf",
+        "tso": "lpar1.tso",
+        "ssh": "lpar1.ssh",
+        "base": "base"
+    },
+    "autoStore": true
+}
+```
+## Accessing LPARs that contain services that do not share the same credentials
+
+In the following configuration, nested profiles use the credentials from different service profiles to access services directly on multiple LPARs.
+
+```
+{
+    "$schema": "./zowe.schema.json",
+    "profiles": {
+        "lpar1": {
+            "properties": {
+                "host": "example1.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 443
+                    }
+                },
+                "tso": {
+                    "type": "tso",
+                    "properties": {
+                        "account": "ACCT#",
+                        "codePage": "1047",
+                        "logonProcedure": "IZUFPROC"
+                    }
+                },
+                "ssh": {
+                    "type": "ssh",
+                    "properties": {
+                        "port": 22
+                    }
+                }
+            },
+            "secure": [
+                "user",
+                "password"
+            ]
+        },
+        "lpar2": {
+            "properties": {
+                "host": "example2.com"
+            },
+            "profiles": {
+                "zosmf": {
+                    "type": "zosmf",
+                    "properties": {
+                        "port": 1443
+                    }
+                }
+            },
+            "secure": [
+                "user",
+                "password"
+            ]
+        },
+        "base": {
+            "type": "base",
+            "properties": {
+                "rejectUnauthorized": true
+            }
+        }
+    },
+    "defaults": {
+        "zosmf": "lpar2.zosmf",
+        "tso": "lpar1.tso",
+        "ssh": "lpar1.ssh",
+        "base": "base"
+    },
+    "autoStore": true
+}
+```
+
+## Accessing LPARs that access services through one API Mediation Layer
+
+In the following configuration, services are accessed through the API ML (where multi-factor authentication (MFA) or single sign-on (SSO) is achievable) using token-based authorization stored in a base profile.
+
+```
+{
+    "$schema": "./zowe.schema.json",
+    "profiles": {
+        "zosmf": {
+            "type": "zosmf",
+            "properties": {
+                "basePath": "api/v1"
```

Comment:
@zFernand0 are these example basePaths right/realistic?

Reply:
This was technically correct in the early days of Zowe, but we should change it to the new standards/recommendations.
For the zosmf profile: `ibmzosmf/api/v1`
For the CICS and DB2 services, I'm afraid we haven't had an official Zowe REST API, but if we do, it would make sense to have the route/path as `ibmcics/api/v1` and `ibmdb2/api/v1` 😋

Reply:
@adam-wolfe @zFernand0 -- should I modify the paths for the CICS and DB2 profiles? (I already updated the base for zosmf)

Reply:
I think it would be good to update them. that way we reflect the new standards 😋

Reply:
There are API ML paths as part of Zowe  

E.g. /gateway/api/v1/version

With Gateway as base path